bundles/org.openhab.binding.ojelectronics/src/main/java/org/openhab/binding/ojelectronics/internal/services/RefreshService.java

```diff
@@ -96,14 +102,47 @@ public void start(String sessionId, BiConsumer<@Nullable GroupContentResponseMod
      */
     public void stop() {
         destroyed = true;
-        final ScheduledFuture<?> scheduler = this.scheduler;
-        if (scheduler != null) {
-            scheduler.cancel(false);
+        final Connection signalRConnection = this.signalRConnection;
```

Comment:
I would prefer a local variable name different from the member name to avoid potential confusions.